nestjs/src/task/task-criteria/task-criteria.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Body, Patch, Param, UseGuards, Post } from '@nestjs/common';
+import { ApiBody, ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import { TaskCriteriaService } from './task-criteria.service';
+import { TaskCriteriaDto } from './dto/task-criteria.dto';
+import { DefaultGuard, RoleGuard } from '../../auth';
+
+@Controller('task/:taskId/criteria')
+@ApiTags('criteria')
+@UseGuards(DefaultGuard, RoleGuard)
+export class TaskCriteriaController {
+  constructor(private readonly taskCriteriaService: TaskCriteriaService) {}
+
+  @Get()
+  @ApiOperation({ operationId: 'getTaskCriteria' })
+  @ApiOkResponse({ type: TaskCriteriaDto })
+  async get(@Param('taskId') taskId: number) {
```

Comment:
Use `@RequiredRoles([ /* roles */ ])` When use `RoleGuard` in controller. Find examples in repository